client/a8c-for-agencies/sections/overview/overview.tsx

```diff
@@ -19,15 +20,19 @@ export default function Overview() {
 	return (
 		<Layout title={ title } wide sidebarNavigation={ <MobileSidebarNavigation /> }>
 			<LayoutTop>
-				<LayoutHeader>
-					<Title>{ title }</Title>
-					<Actions>
-						<OverviewHeaderActions />
-					</Actions>
-				</LayoutHeader>
+				<div className="a4a-overview-header">
```

Comment:
[Non-blocker] Can we modify the `LayoutHeader` to support the class name instead of adding a div here?

Reply:
Sure! Thanks!